README.md

```diff
@@ -53,16 +56,25 @@ Setting up a Heroku pipeline is relatively straightforward. We set up a pipeline
 3. Follow the Create an Application steps in [this documentation](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_Ruby_sinatra.html#create_deploy_Ruby_eb_init).
-  * For steps 5/6, select `Ruby` and `Ruby 2.2 (Puma)`
+  * For steps 5/6, select `Ruby` and `Ruby 2.5 (Puma)`
+  * NOTE: AWS config only works on specific versions of Ruby. We went with 2.5.3. 
   * SSH login is optional, but convenient
 4. At this point, you'll want to set up the DB. We created an integrated Postgres database instance (v. 9.5.2) as described in [here](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/using-features.managing.db.html).
-5. Walk through [Create an Environment](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_Ruby_sinatra.html#create_deploy_Ruby_eb_env)
-6. ‼️ At this point, stop and check on the instance type. You may need to configure a VPC.
-7. Try deploying: `eb deploy`
-
-Configuring the "email to yourself" feature requires extra configuration on EB.
+  * You will need to configure Amazon RDS. Create a database and use the information from the DB to connect (this is important to do first before you upload and deploy)
+  * An AWS RDS database will follow the same Database URL as the given format in `.ENV` as if you were running Postgres locally. You can find the necessary credentials in the RDS dashboard for your database instance.
+  * When you are using AWS RDS, you might have to connect from outside to run migrations. To do this, go to your database instance, add a new inbound security rule for "Anywhere" IP, and then run your migrations locally/remotely, then remove the inbound security role.
+5. Create an S3 Bucket for use in storing uploaded Resume files from the Job System form. Copying bucket policies from the Elastic Beanstalk auto-generated bucket works fine from what we've seen, or you can set specific permissions as needed.
+6. Walk through [Create an Environment](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_Ruby_sinatra.html#create_deploy_Ruby_eb_env)
+7. Create a source bundle if uploading through the UI. `zip ../filename.zip -r * .[^.]*`
+8. ‼️ At this point, stop and check on the instance type. You may need to configure a VPC.
+9. Try deploying: `eb deploy` or use the Upload and Deploy feature integrated into EB.
+  * If the app is not working, try using a different Ruby version (e.g. 2.5.1). Note that this would mean testing to ensure all the Ruby gems work, especially if downgrading.
+
+~~Configuring the "email to yourself" feature requires extra configuration on EB.~~
+If you are using AWS SES SMTP service, this is no longer necessary as the emails will go through SES. We have set up an SMTP service, so all emails are going through a central location (AWS)
```

Comment:
@tmloupe Is this true? I think there's still some pony-based mail happening in the app ([here](https://github.com/loyno-mathcs/workforwardnola/blob/master/app.rb#L195-L210)).

Reply:
I will clarify what is going on here - thanks for catching. We are still going to use the pony gem there since it is integrated (and works without a lengthy POST method) but it is configured using SMTP through the ENV file.